src/routes/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Header from '../components/Header';
+import HomeView from '../views/HomeView';
+import LoginView from '../views/LoginView';
+
+const Routes = () => (
+    <Router>
+        <div>
```

Comment:
Great work you've done here @codjoero.
Just on this Router functional component, I wanted to as is the Login page also going to have a Header (Navbar) or should this be this be conditional and appear on pages like home, profile among others.
I mean based off of what Vicky did in the mockups.